.github/workflows/validate-models.yml

```diff
@@ -0,0 +1,147 @@
+---
+name: Validate OpenSCAD Models
+
+"on":
+  push:
+    branches: [main, develop]
+    paths:
+      - 'models/**/*.scad'
+      - '.github/workflows/validate-models.yml'
+  pull_request:
+    paths:
+      - 'models/**/*.scad'
+      - '.github/workflows/validate-models.yml'
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+
+    env:
```

Comment:
I'd rather the source of truth is from the install scripts. no need here for versions.